docs/UserGuide.md

```diff
@@ -170,7 +170,7 @@ In the example above , `find` is the **command word** while `n/` is the **prefix
 | NAME      | n/     | Show name must be **unique** and contains only [**alphanumeric**](#glossary) characters.<br>Example: If you try to add `n/Stranger Things` into the show list that already contains that show, there will be a message telling you that this show already exists in the list. You can name it as `n/Stranger Things S2` instead. |
 | STATUS    | s/     | Status can only contain _COMPLETED_, _WATCHING_ and _PLAN-TO-WATCH_                                                                                                                                                                                                                                               |
 | RATING    | r/     | Rating must be a **whole number** from **0 to 5**.                                                                                                                                                                                                                                                                    |
-| COMMENT   | c/     | A comment can contain **any text**, but **emojis will be removed** from input.                                                                                                                                                                                                                                                                                                                      |
+| COMMENT   | c/     | A comment can only contain **ASCII values** and is limited to 300000 **ASCII values**.                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Maybe we can add a link to the glossary table for `ASCII`?
Also maybe we could rephrase it to `A comment can only contain up to 300000 ASCII values` instead?